core-common/src/main/java/org/zalando/nakadi/repository/kafka/KafkaRecordDeserializer.java

```diff
@@ -55,6 +57,18 @@ public KafkaRecordDeserializer(final AvroSchema schemas) {
             metadataParser.setSchema(new com.fasterxml.jackson.dataformat.avro.
                     AvroSchema(schemas.getMetadataSchema()));
             final ObjectNode metadataNode = metadataParser.readValueAsTree();
+            metadataNode.put(
+                    "occurred_at",
+                    new DateTime(
+                            metadataNode.get("occurred_at").asLong(),
```

Comment:
Why this conversion from long to date here?

Reply:
bugfix